boards/px4/fmu-v5x/src/board_config.h

```diff
@@ -250,8 +250,8 @@
 #define GPIO_VDD_3V3_SD_CARD_EN         /* PC13 */ (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_CLEAR|GPIO_PORTC|GPIO_PIN13)
 
 /* MCP23009 GPIO expander */
-#define BOARD_GPIO_VDD_5V_COMP_VALID           "/dev/gpin4"
-#define BOARD_GPIO_VDD_5V_CAN1_GPS1_VALID      "/dev/gpin5"
```

Comment:
Well, well, well… I was wondering why my driver worked in firmware_private, but not in upstream.

Turns out, the pins are actually registered at `/dev/gpio4` and `/dev/gpio5`. So the function [`ADC::open_gpio_devices()`](https://github.com/PX4/PX4-Autopilot/blob/fc357921fd69e574a3f596be25faa8f4d9cdcea9/src/drivers/adc/board_adc/ADC.cpp#L127) always got `-1` as file handles and thus [never actually called the MCP23009 driver](https://github.com/PX4/PX4-Autopilot/blob/fc357921fd69e574a3f596be25faa8f4d9cdcea9/src/drivers/adc/board_adc/ADC.cpp#L180). 

This was working when the driver was added (e2337a34eb1bb59e4befc1e197cb8a7359e35041), but since then [NuttX changed the format string to generic `gpio`](https://github.com/PX4/NuttX/commit/2e47ef32cf850bdf604d49fd44bc3c5ca313bcee).

So since then, apparently nobody ever noticed this stopped working… 🫠
Fortunately these are the only occurances of `/dev/gpin`, `/dev/gpint`, and `/dev/gpout` in PX4.

We should be careful when we upgrade NuttX in firmware_private.